filebeat/docs/multiline.asciidoc

```diff
@@ -46,10 +46,9 @@ To consolidate these lines into a single event in Filebeat, use the following mu
 
 [source,json]
 -------------------------------------------------------------------------------------
-multiline:
-  pattern: '^[[:space:]]'
-  negate: false
-  match: after
+multiline.pattern: '^[[:space:]]'
```

Comment:
I was thinking if we should have the "correct" indentation for these sub fields, so if someone does a copy paste it is already with the right indentation?